config/StoredProcedures/EditUserByUsername

```diff
@@ -0,0 +1,15 @@
+use imp_db_dev;
```

Comment:
Ok I pulled the most recent version of master w/ your SQL changes. Was there anything else?